docs/sources/administration/configuration.md

```diff
@@ -772,6 +772,22 @@ Refer to [Auth proxy authentication]({{< relref "../auth/auth-proxy.md" >}}) for
 
 Refer to [LDAP authentication]({{< relref "../auth/ldap.md" >}}) for detailed instructions.
 
+## [aws]
+
+You can configure core and external AWS plugins.
+
+### allowed_auth_providers
+
+Specify what authentication providers that the AWS plugins allow. For a list of allowed providers, refer to the data-source configuration page for a given plugin. If you configure a plugin by provisioning, only providers that are specified in `allowed_auth_providers` are allowed.
+
+Options: `default` (AWS SDK Default), `keys` (Access && key secret key), `credentials` (Credentials field), `ec2_IAM_role` (EC2 IAM Role)
```

Comment:
```suggestion
Options: `default` (AWS SDK default), `keys` (Access  and key secret key), `credentials` (Credentials field), `ec2_IAM_role` (EC2 IAM role)
```
Why are there two instances of the word "key" in "key secret key"?

Where do these options appear?

Reply:
oh that's a typo! they appear in the auth drop down of the data source config page. You can check the CloudWatch config page for example.

Reply:
I assume there is more than one key: access key and secret key.